generic-ec/src/non_zero/mod.rs

```diff
@@ -296,6 +316,57 @@ impl<E: Curve> AsRef<Scalar<E>> for NonZero<SecretScalar<E>> {
     }
 }
 
+impl<T> cmp::PartialEq<T> for NonZero<T>
```

Comment:
We don't have `PartialEq<Self> for NonZero<T>`, or do we?

Reply:
We do, it's autoderived